### PR TITLE
Change Model type to `OrderedCollectionPage`

### PR DESCRIPTION
**Pull request by kPherox**
Use type `OrderedCollectionPage` instead of `OrderedCollection` in pages or distinct subsets
FYI: https://www.w3.org/TR/activitystreams-core/#h-paging

### CODE REVIEW

File: src/Protocol/ActivityPub/Transmitter.php [hunk 1]
```diff
@@ -159,12 +161,13 @@ public static function getOutbox($owner, $page = null)
 
 		$data = ['@context' => ActivityPub::CONTEXT];
 		$data['id'] = System::baseUrl() . '/outbox/' . $owner['nickname'];
-		$data['type'] = 'OrderedCollection';
```
Comment (MrPetovan) [inline src/Protocol/ActivityPub/Transmitter.php]: Can you please decide on one consistent way of enabling your change, either by moving all the types declarations to the  `if (empty($page))` conditions or leaving all default type declaration out of them?

**Comment by annando** [comment]
@kPherox thanks for your work! Indeed some time ago I read some discussion about the fact that forgot some value in our outbox. I totally forgot about this later on.

File: src/Protocol/ActivityPub/Transmitter.php (hunk 1 above)
Comment (kPherox) [reply src/Protocol/ActivityPub/Transmitter.php]: I was to move to the `empty($page)` conditions. 0c732f2

Comment (kPherox) [reply src/Protocol/ActivityPub/Transmitter.php]: Sorry, I forgot the `! empty ($ profile ['hide-friends'])` conditions. I will to use the latter method.

**Pull request merged**